CHANGELOG.md

```diff
@@ -3,6 +3,9 @@
 ## Master (Unreleased)
 
 * Fix an error for `RSpec/Capybara/SpecificFinders` with no parentheses. ([@ydah][])
+* Add support for subject! method to `RSpec/SubjectDeclaration`. ([@ydah][])
```

Comment:
This comes from a different branch, doesn't it?

Reply:
Oops, I delete it.